share/spice/rand_word/rand_word.js

```diff
@@ -31,6 +31,20 @@
                     moreAt: true
                 }
             }
-        });
+        };
+        
+        if ( list_of_list.length == 1 ) {
+            spiceObj.data = list_of_list[0];
```

Comment:
Let's capture the first item in `api_result` above, before we start splicing the list, so that we can easily access it here:

```js
var first = api_result[0];
...

if ( list_of_list.length == 1 ) {
    spiceObj.data = {
        title: "Random Word",
        word: first.word
    };
    spiceObj.templates = {
         group: 'text',
        options: {
            content: Spice.rand_word.single_word,
             moreAt: true
        }
    };
}
```